canonical-web-and-design/maas-ui
# WIP: Debounce machine filtering.

## Done
- Debounce machine filter function so that it only runs if there's been at least a 500ms interval. In the meantime, a spinner is displayed.

## QA
- Type in the filter box and check that a spinner shows up immediately.
- Check that the machine list doesn't update until you stop typing for at least 500ms.
